rosidl_typesupport_introspection_c/CMakeLists.txt

```diff
@@ -11,7 +11,10 @@ find_package(ament_cmake_python REQUIRED)
 
 ament_export_dependencies(rosidl_cmake)
 ament_export_dependencies(rosidl_generator_c)
-ament_export_include_directories(include)
+# The reason the impl folder is exported is that it contains the implementation
+# for the get_*_type_support_handle functions and defines the opensplice
```

Comment:
Is this comment correct? It seems to be specific to the Opensplice typesupport package.